docs/_static/mxnet-theme/index.html

```diff
@@ -21,21 +21,21 @@
   <div class="container">
     <div class="row">
       <div class="col-lg-4 col-sm-12">
-        <h3>Apache MXNet 1.1.0 Released</h3>
-        <p>We're excited to announce the release of MXNet 1.1.0! Check out the release notes for latest updates.</p>
-        <a href="https://github.com/apache/incubator-mxnet/releases/tag/1.1.0">Learn More</a>
+        <h3>MXNet 1.2.0.rc0 Released</h3>
+        <p>We're excited to announce the release of MXNet 1.2.0.rc0! Check out the release notes for latest updates.</p>
+        <a href="https://github.com/apache/incubator-mxnet/releases/tag/1.2.0">Learn More</a>
```

Comment:
The link should be : `https://github.com/apache/incubator-mxnet/releases/tag/1.2.0`

Reply:
Ok, thanks for the feedback. I will change this.